<launch>
       <arg name="robot_name" default="alpha_rise"/>
       <node ns="$(arg robot_name)" pkg="pcl_proc" type="wp_admin.py" name = "waypoint_administrator" output="screen"/>
</launch>